master.sh

```diff
@@ -40,13 +40,7 @@ init() {
 	fi
 
 	# Set working mode
-	if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
-		echo "Start k8s cluster in installation mode"
 		RESTART_POLICY="always"
```

Comment:
Ok let me explain. With your change, after reboot, k8s is running independent of the mode "run" or "install". Only implementation detail (the way it is installed) changed. 

My intention was rather to support two use cases: 1. the user wants to run k8s without installation. 2. the user wants to install k8s. The reboot behavior should be set accordingly

Makes sense?